src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -210,6 +215,18 @@ namespace CommandLineInterface {
 
         bool eplusRunningViaAPI = state.dataGlobal->eplusRunningViaAPI;
 
+        opt.get("-j")->getInt(state.dataGlobal->numThread);
+
+        if (state.dataGlobal->numThread == 0) {
+            DisplayString(state, "Invalid value for -j arg. Defaulting to 1.");
+            state.dataGlobal->numThread = 1;
+        } else if (state.dataGlobal->numThread > (int)std::thread::hardware_concurrency()) {
```

Comment:
Interesting.  I'm not sure we really need to take the burden of restricting this.  And on some platforms where this cannot be determined, it is required to return 0, thus making EnergyPlus fail.  I think I'll take this out.